org.eclipse.scanning.malcolm.core/src/org/eclipse/scanning/malcolm/core/AbstractMalcolmDevice.java

```diff
@@ -84,6 +84,7 @@ public AbstractMalcolmDevice(IMalcolmConnectorService<MalcolmMessage> connector,
 	public void setPointGenerator(IPointGenerator<?> pointGenerator) {
 		this.pointGenerator = pointGenerator;
 	}
+
 	public IPointGenerator<?> getPointGenerator() {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Remove usage of generic wildcard type. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1452)